docs/sources/profiling-types-in-pyroscope/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Understanding Profiling Types and Their Uses in Pyroscope"
+menuTitle: "Understanding Profiling Types in Pyroscope"
+description: "Learn about the different profiling types available in Pyroscope and how to effectively use them in your application performance analysis."
```

Comment:
Small stylistic things -- we use [sentence case](https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/) instead of title case. It's also not necessary to use quotes in the front matter of the markdown files. For more information, see https://grafana.com/docs/writers-toolkit/write/front-matter/

```suggestion
title: Understanding profiling types and their uses in Pyroscope
menuTitle: Understanding profiling types
description: Learn about the different profiling types available in Pyroscope and how to effectively use them in your application performance analysis.
```